### PR TITLE
bottom_position

### PR DESCRIPTION
**Pull request by mattpake**
Added new parameter for height multiplier

### CODE REVIEW

**Comment by mattpake** [comment]
https://cesko-digital.atlassian.net/browse/LOON-396

**Comment by mattpake** [comment]
https://cesko-digital.atlassian.net/browse/LOON-396

File: lib/ui/widgets/onboarding/carousel/button.dart [hunk 1]
```diff
@@ -2,15 +2,16 @@ import 'package:flutter/material.dart';
 import 'package:loono/ui/widgets/button.dart';
 
 class CarouselButton extends StatelessWidget {
-  const CarouselButton({Key? key, required this.text, this.onTap}) : super(key: key);
+  const CarouselButton({Key? key, required this.heightMultiplier, required this.text, this.onTap}) : super(key: key);
```
Comment (mzdm) [inline lib/ui/widgets/onboarding/carousel/button.dart:5]: Just a minor thing. Could you please apply formatting here? We use `100 line length`, if you need help let me know

Comment (mattpake) [reply lib/ui/widgets/onboarding/carousel/button.dart:5]: Normally I used auto formatting OPTION + COMMAND + L . Here in this project is in most classes not used WHY? There should be generally rules for everyone .

Comment (mzdm) [reply lib/ui/widgets/onboarding/carousel/button.dart]: resolved in chat

**Review by mzdm — APPROVED** [review]
tested on small resolution, lgtm

**Pull request merged**